waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -55,10 +55,13 @@ genWebApp spec = do
   sequence
     [ genFileCopy [relfile|README.md|],
       genFileCopy [relfile|tsconfig.json|],
+      genFileCopy [relfile|tsconfig.node.json|],
       genPackageJson spec (npmDepsForWasp spec),
       genNpmrc,
       genGitignore,
-      return $ C.mkTmplFd $ C.asTmplFile [relfile|netlify.toml|]
+      return $ C.mkTmplFd $ C.asTmplFile [relfile|netlify.toml|],
```

Comment:
Why not just `genFileCopy`? Also, could we group all these guys together (at the start)?